### PR TITLE
Changes based on comments by Harald A (pdf file)

### PR DESCRIPTION
**Pull request by cdh4u**

### CODE REVIEW

File: draft-ietf-ice-rfc5245bis.xml [hunk 1]
```diff
@@ -678,9 +678,9 @@ responding agent, the peer is the initiating agent. </t>
 transport protocol (such as UDP or TCP) port.</t>
 
 <t hangText="Media, Media Stream, Media Session:"> When ICE is used to
-setup multimedia sessions, the media is usually transported over RTP,
+setup communication sessions, the media is usually transported over RTP,
```
Comment (alvestrand) [inline draft-ietf-ice-rfc5245bis.xml]: For this particular instance, it should stay "multimedia", I think.

File: draft-ietf-ice-rfc5245bis.xml [hunk 2]
```diff
@@ -678,9 +678,9 @@ responding agent, the peer is the initiating agent. </t>
...
 and a media stream composes of a stream of RTP packets. When ICE is
-used with other than multimedia sessions, the terms "media", "media
+used with other than communication sessions, the terms "media", "media
```
Comment (alvestrand) [inline draft-ietf-ice-rfc5245bis.xml]: And here.

**Comment by alvestrand** [comment]
LGTM, modulo the two comments above.

File: draft-ietf-ice-rfc5245bis.xml (hunk 1 above)
Comment (cdh4u) [reply draft-ietf-ice-rfc5245bis.xml]: Ok.

File: draft-ietf-ice-rfc5245bis.xml (hunk 2 above)
Comment (cdh4u) [reply draft-ietf-ice-rfc5245bis.xml]: Ok.

**Pull request merged**